contents/docs/migrate/migrate-to-cloud.mdx

```diff
@@ -4,7 +4,11 @@ sidebar: Docs
 showTitle: true
 ---
 
-> If you're attempting this migration, feel free to ask questions and provide feedback via [raising a support ticket in the app](https://app.posthog.com/home#supportModal). 
+> Prior to beginning a data migration please ensure that you've done the following:
+> 1. Create a project on our [US](https://us.posthog.com/signup) or [EU](https://eu.posthog.com/signup) Cloud
+> 2. Sign up to a paid plan on the billing page in-app (historic imports are not billable but this will unlock the necessary features).
```

Comment:
```suggestion
> 2. Sign [up to a paid plan on the billing page](https://us.posthog.com/organization/billing) (historic imports are not billable but this will unlock the necessary features).
```